PendingReleaseNotes

```diff
@@ -396,3 +396,8 @@ Relevant tracker: https://tracker.ceph.com/issues/57090
 set using the `fs set` command. This flag prevents using a standby for another
 file system (join_fs = X) when standby for the current filesystem is not available.
 Relevant tracker: https://tracker.ceph.com/issues/61599
+* mon: add NVMe-oF gateway monitor and HA
+  adding 2 new mon commands:
+    - nvme-gw create
+    - nvme-gw delete
```

Comment:
Please add a one or two sentence summary for what the gateway monitor is. What do the commands do?